spec/operators/publishReplay-spec.ts

```diff
@@ -438,8 +438,7 @@ describe('publishReplay operator', () => {
     const source = cold('--1-2---3-4---|');
     const published = source.pipe(publishReplay(1, Infinity, selector));
 
-    // The exception is thrown outside Rx chain (not as an error notification).
-    expect(() => published.subscribe()).to.throw(error);
+    expectObservable(published).toBe('#', undefined, "It's broken");
```

Comment:
Here, there's a strange, old test asserting that errors thrown in a selector function provided to `publishReplay` -- and therefor, by proxy, `multicast`, `publish`, et al -- are simply "thrown" into the ether. Which is obviously wrong.

When `multicast` got the unified operator creation treatment with the new `operate` function, any error thrown during the initialization of the subscription created within the multicast operation was caught and sent to the consumer via an error notification. As we always should have been doing.